ipaserver/install/ipa_otptoken_import.py

```diff
@@ -106,7 +107,21 @@ def convertHashName(value):
         "sha-256": u"sha256",
         "sha-384": u"sha384",
         "sha-512": u"sha512",
-    }.get(value.lower(), u"sha1")
+    }
+
+    v = value.lower()
+    for prefix in known_prefixes:
+        if len(prefix) > 0:
```

Comment:
Just a nit, I think this is equivalent and more efficient:

if prefix:

Reply:
Right, this is just cleaner as well. I've pushed an update.